cli/chain.go

```diff
@@ -1504,3 +1511,298 @@ var ChainPruneCmd = &cli.Command{
 		return api.ChainPrune(ctx, opts)
 	},
 }
+
+// TODO: Find a home for this.
```

Comment:
Yeah, we have this in 7 different places, we should resolve this before merging.

Reply:
Pretty sure that there's a bug here anyway.